pages/careers/_human-friendly.html

```diff
@@ -19,15 +19,17 @@ <h3>How We Work</h3>
     <h3>Benefits</h3>
     <p>Our benefits reflect our values. We believe compensation should be fair, transparent, and generous. We hire Grunts in many countries, so some details may vary.</p>
     <ul>
-      <li><strong>Above-Market Salary.</strong> To reduce bias and increase transparency, we compute all salaries <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">using formulas</a>. The formula factors in your title and location and uses a multiplier to produce a result that's above market for that title and location.</li>
+      <li><strong>Location Independent Above-Market Salary.</strong> To reduce bias and increase transparency, we compute all salaries <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">using formulas</a>. The formula factors in your title and uses a multiplier to produce a result that's above market for that title. Our salaries are location independent.</li>
       <li><strong>Profit-Sharing Bonus.</strong> We set aside a pot of money at the end of each year based on profits and distribute bonuses according to a formula that uses as inputs your level within the company and the length of your tenure at the company.</li>
-      <li><strong>Performance Bonus.</strong> We give performance-based bonuses as often as once a quarter, depending on your performance.</li>
+      <li><strong>Hardware budget.</strong> We'll buy you a brand new 16" Apple MacBook Pro (or other computer of your choosing of equivalent value) upon joining. It will be owned by you, not the company.</li>
+      <li><strong>Personal Budget.</strong> We'll give you a personal budget of 1000 USD per month to spend on your workspace (e.g., a co-working space), health (e.g., gym, yoga), time (e.g., babysitter), and/or learning (e.g., books, courses).</li>
+    </ul>
+    <p>In addition to the global benefits listed above, we have some US-specific benefits as well:</p>
+    <ul>
       <li><strong>Medical/Dental/Vision Insurance.</strong> We offer a range of high-quality plans with a large portion paid by the company.</li>
       <li><strong>FSA and HSAs.</strong> We don't contribute to these accounts, but we do offer them as an option.</li>
       <li><strong>Pension/401(k).</strong> We contribute a portion of your salary to your pension or 401(k).</li>
```

Comment:
We contribute to everyone's pension; it's just that it's optional in the USA! Would you mind moving that above the US-specific label?

Reply:
I see what you're saying, however to me this reads as Gruntwork contributes to a private pension plan (which would be on top of statutory pension), which is not the case for any of the non-US countries if I'm reading the Benefits page right. Speaking for the EU at least, the statutory pension is guaranteed by law, so it's not customary to call that out in a job post. I propose that we leave this item under US-specific section, but remove mention of "pension" and leave only 401K. That way the US candidates now that they get 401K, and non-US candidates don't get the false impression that they will get private pension contributions as a benefit. What do you think?

Reply:
Ah, that sounds perfect, and thank you for helping me understand convention around the EU!

Reply:
Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/684/commits/593ef439f8119da4388845212a1687d1b0e24177.